include/spirv-tools/libspirv.hpp

```diff
@@ -82,17 +82,26 @@ class SpirvTools {
   // Returns true on successful assembling. |binary| will be kept untouched if
   // assembling is unsuccessful.
   bool Assemble(const std::string& text, std::vector<uint32_t>* binary) const;
+  // |text_size| specifies the number of bytes in |text|.
+  bool Assemble(const char* text, size_t text_size,
```

Comment:
Added comments.

Actually this is to be consistent with `spvTextToBinary`, which takes a size parameter. Should we do normal null-terminating instead? It's kinda safe to require a size, but less conventional.